Dockerfile.rhel

```diff
@@ -2,7 +2,8 @@ FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
 
 WORKDIR /go/src/github.com/prometheus/node_exporter
 COPY . .
-RUN make build
+ENV BUILD_PROMU=false
+RUN yum install -y prometheus-promu && make build && yum clean all
```

Comment:
It's probably not even necessary to "yum clean all" since promu is only installed in the build image, and not in the runtime image, but I'm not sure what is the best practice here.

Reply:
I was unsure too so in doubt I kept it.

Reply:
yum clean in a builder is not necessary but won't really hurt anything.